### PR TITLE
Update Makefile to be more quiet

### PR DESCRIPTION
**Pull request by jgrund**
The makefile is spewing out lots of extra info which seems to confuse
the build process. Make it more quiet.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,8 +23,19 @@ jobs:
...
+      script:
+        - export OWNER=managerforlustre
+        - export PROJECT=manager-for-lustre-devel
+        - export PACKAGE=iml-view-server
+        - export CLONE_URL=https://github.com/whamcloud/view-server.git
+        - export SPEC=iml-view-server.spec
+        - docker run -it -e OWNER="$OWNER" -e PROJECT="$PROJECT" -e PACKAGE="$PACKAGE" -e CLONE_URL="$CLONE_URL" -e SPEC="$SPEC" -e KEY="$encrypted_253525cedcf6_key" -e IV="$encrypted_253525cedcf6_iv" -v $(pwd):/build:rw imlteam/copr
     - stage: deploy-copr
       name: "Copr deploy"
+      language: generic
```
Comment (johnsonw) [inline .travis.yml]: Do we need a language property here?

Comment (jgrund) [reply .travis.yml]: Not sure what you mean, there is one there.

Comment (johnsonw) [reply .travis.yml]: right, just asking if it's necessary?

Comment (jgrund) [reply .travis.yml]: Yes, it will choose ruby if you don't specify generic iirc.

Comment (johnsonw) [reply .travis.yml]: 👍

**Pull request merged**